<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="lbLanguage">
           <property name="text">
            <string>Language</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cbLanguage"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbDateFormat">
           <property name="text">
            <string>Date format</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" rowspan="2">
          <widget class="QLineEdit" name="leDateFormat"/>
         </item>
         <item row="2" column="0" rowspan="2">
          <widget class="QLabel" name="lbTimeFormat">
           <property name="text">
            <string>Time format</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="leTimeFormat"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="cbUseSystemDateTimeFormat">
         <property name="text">
          <string>Use system date/time format</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbOpenLastFilesAtStartup">
         <property name="text">
          <string>Open last files at startup</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacerGeneral">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>131</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabColumns">
      <attribute name="title">
       <string>Columns</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lbVisibleColumns">
         <property name="text">
          <string>Visible</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="lbAvailableColumns">
         <property name="text">
          <string>Available</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QListWidget" name="lwVisibleColumns">
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QToolButton" name="btnAddCol">
           <property name="text">
            <string notr="true">&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnDelCol">
           <property name="text">
            <string notr="true">&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnUpCol">
           <property name="text">
            <string notr="true">É…</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnDownCol">
           <property name="text">
            <string notr="true">V</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <widget class="QListWidget" name="lwAvailableColumns">
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSaving">
      <attribute name="title">
       <string>Saving</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lbPrefVCardVer">
           <property name="text">
            <string>Preferred vCard version</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbPrefVCardVer">
           <item>
            <property name="text">
             <string>2.1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3.0</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="cbUseOrigVer">
         <property name="text">
          <string>Use original file version (if present)</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lbDefaultCountryRules">
           <property name="text">
            <string>Default country</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbDefaultCountryRules"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lbDefaultCountryRules2">
         <property name="text">
          <string>(for phone number internationalization)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbSkipTimeFromDate">
         <property name="text">
          <string>Skip time from birthday and anniversaries</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>146</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLoading">
      <attribute name="title">
       <string>Loading</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lbDefaultEmptyPhoneType">
           <property name="text">
            <string>Default type if empty</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbDefaultEmptyPhoneType"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lbDefaultPhoneHint">
         <property name="text">
          <string>(many phones treats 'Voice' as 'Other')</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbWarnOnNonStandardTypes">
         <property name="text">
          <string>Warning on non-standard phone/email types</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>150</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
